integration/token/fungible/views/redeem.go

```diff
@@ -84,7 +84,7 @@ func (t *RedeemView) Call(context view.Context) (interface{}, error) {
 	net := network.GetInstance(context, tx.Network(), tx.Channel())
 	vault, err := net.Vault(tx.Namespace())
 	assert.NoError(err, "failed to retrieve vault [%s]", tx.Namespace())
-	vc, err := vault.Status(tx.ID())
+	vc, _, err := vault.Status(tx.ID())
```

Comment:
We can remove the second output param from `Status`, as it seems it isn't used anywhere and the signature is not part of an interface.